src/scss/_slider-input.scss

```diff
@@ -1,51 +1,104 @@
-$slider-handle-size: 2.2rem;
+$thumb-size: 20px;
+$thumb-color: $white;
+$thumb-focus-shadow: 0 0 8px $blue;
+$track-height: 4px;
+$track-color: $blue;
 
-.slider-input {
-  display: flex;
-  align-items: center;
+@mixin thumb {
```

Comment:
Usually I'd say to move this to the `mixins/` dir but since this is and will only ever be used here I'd say it's fine... 
Might be nice to rename it to something more inline with the file (ie `slider-input-thumb` or similar).